vitrina/datasets/forms.py

```diff
@@ -1,7 +1,50 @@
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import Div, Field, Submit, Layout
+from django import forms
+from .models import Dataset
+from django.utils.translation import gettext_lazy as _
 from django.forms import Form, CharField, DateTimeField, IntegerField
-
 from vitrina.fields import MultipleValueField, MultipleIntField
 
+class NewDatasetForm(forms.ModelForm):
+    class Meta:
+        model = Dataset
+        fields = ('is_public', 'title', 'description', 'tags',
+                  'category', 'licence', 'update_frequency',
+                  'access_rights', 'distribution_conditions', 'manager',
+                  'representative_id')
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.helper = FormHelper()
+        self.helper.form_id = "dataset-form"
+        self.helper.layout = Layout(
+            Div(Div(Field('is_public', css_class='input', placeholder=_('Ar duomenys vieši')),
+                    css_class='control'), css_class='field'),
```

Comment:
This field should be shown as checkbox, now it is shown as drop down list with „Unknow“, „Yes“, „No“. This is because `Datset.is_public` has a `null=True`, but it shouldn't.